inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/KnowledgeBaseServiceImpl.java

```diff
@@ -817,6 +825,7 @@ public KBHandle update(KnowledgeBase kb, UpdateAction aAction)
                 tupleQuery.setBinding("pLABEL", aKB.getLabelIri());
                 tupleQuery.setBinding("pDESCRIPTION", aKB.getDescriptionIri());
                 tupleQuery.setIncludeInferred(false);
+                tupleQuery.setMaxExecutionTime(0);
```

Comment:
I believe that `0` is actually the default (i.e. no timeout). Mind that on remote SPARQL connections, this setting has no effect. What are you trying to achieve?

Reply:
Yeah. Its the default one. Its still WIP. I was checking out some things with the tuple query.